include/rawpheno.backup.form.inc

```diff
@@ -165,7 +165,10 @@ function rawpheno_backup($form, &$form_state, $project_id = NULL, $command = 'de
         '#file_upload_max_size' => '10M',
         '#upload_location' => 'public://',
         '#upload_event' => 'manual',
-        '#upload_validators' => array('file_validate_extensions' => array('xlsx xls')),
+        '#upload_validators' => array(
+          'file_validate_extensions' => array('xlsx xls'),
+          'rawpheno_validate_spreadsheet_file' => array($form_state),
```

Comment:
This is an interesting approach I hadn't thought of --good idea! 
I think for file validators the first argument provided to the function is always $file and you only need to pass in arguments above if you need more then that. Thus you should be able to pass in nothing here: `'rawpheno_validate_spreadsheet_file' => array(),` and still use `function rawpheno_validate_spreadsheet_file($file) { }` below.